docs/tutorial/flink.md

```diff
@@ -1,94 +1,407 @@
+   c.InputCacheStorage.root_path = f"{BUCKET_PREFIX}/cache/input"
+   ```
+
+
+### Other Configuration Options
+
+A [subset of the configuration schema](https://github.com/pangeo-forge/pangeo-forge-recipes/blob/main/pangeo_forge_recipes/injections.py) 
+that we just talked about related to inputs/outputs/caches gets dependency injected into the recipe by the runner. 
+
+Various other runner options ([documented here](https://pangeo-forge-runner.readthedocs.io/en/latest/reference/index.html) and [flink-specific here](https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/deployment/config/)) 
+can also be passed either in the file configuration format above or directly during CLI `bake` calls. 
 
-You can add `--prune` if you want to only test the recipe and run just the first
-few steps.
+Here's a quick example of something slightly more complicated where you're passing [flink-specific configuration options](https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/deployment/config/) and runner options. 
+Note that `-f <path-to-your-runner-config>.<json||py>` would point to our JSON or `traitlet` runner configuration file we just talked about above:
 
-This might take a minute to submit.
+   ```bash
+   pangeo-forge-runner bake \
+       --repo="https://github.com/pforgetest/gpcp-from-gcs-feedstock.git"  \
+       --ref="0.10.3" \
+       -f <path-to-your-runner-config>.<json||py> \
+       --FlinkOperatorBakery.job_manager_resources='{"memory": "2048m", "cpu": 1.0}' \
+       --FlinkOperatorBakery.task_manager_resources='{"memory": "2048m", "cpu": 1.0}' \
+       --FlinkOperatorBakery.flink_configuration='{"taskmanager.numberOfTaskSlots": "1", "taskmanager.memory.flink.size": "1536m", "taskmanager.memory.task.off-heap.size": "256m", "taskmanager.memory.jvm-overhead.max": "1024m"}' \
+       --FlinkOperatorBakery.parallelism=1 \
+       --FlinkOperatorBakery.flink_version="1.16" \
+       --Bake.prune=True \
+       --Bake.job_name=recipe \
+       --Bake.container_image='apache/beam_python3.9_sdk:2.50.0' \
+       --Bake.bakery_class="pangeo_forge_runner.bakery.flink.FlinkOperatorBakery"
+   ```
 
-## Access the Flink Dashboard
+Where you put things is your choice but _please be careful_: you don't want to commit AWS secrets into GH!
 
-After you run the `pangeo-forge-runner` command, amongst the many lines of output,
+## Running the Recipe
+
+Now let's run the [integration test recipe](https://github.com/pforgetest/gpcp-from-gcs-feedstock.git)!
+Below is the minimal required args for running something successful for Flink on `pangeo-forge-runner>=0.9.1`
+where `<path-to-your-runner-config>.<json||py>` is your configuration file talked about above:
+
+   ```bash
+   pangeo-forge-runner bake \
+       --repo=https://github.com/pforgetest/gpcp-from-gcs-feedstock.git  \
+       --ref="0.10.3" \
+       -f <path-to-your-runner-config>.<json||py>
```

Comment:
Same comment as above for putting things into config.

Once you get the config setup, you can mostly completely ignore it and run this against various repos and refs!

Reply:
You can also pass `--prune` instead of `--Bake.prune=True`.

Reply:
yep, changed